### ISSUE TITLE
Invalid index error on list_details.py

### ISSUE THREAD

**Issue by whirm** [Tribler/tribler#1620]
This is on windows 8.1

```
Traceback (most recent call last):
  File "Tribler\Main\vwxGUI\SearchGridManager.pyo", line 606, in _do_gui_callback
  File "Tribler\Main\vwxGUI\list_details.pyo", line 935, in OnRefresh
  File "Tribler\Main\vwxGUI\__init__.pyo", line 119, in invoke_func
  File "Tribler\Main\vwxGUI\list_details.pyo", line 1298, in _Refresh
  File "wx\_controls.pyo", line 4594, in SetItemColumnImage
PyAssertionError: C++ assertion "id >= 0 && id < GetItemCount()" failed at ..\..\src\msw\listctrl.cpp(771) in wxListCtrl::SetItem(): invalid item index in SetItem
```

**Comment by LipuFei** [comment]
@whirm This problem should be fixed. Could you check again?

**Comment by whirm** [comment]
This comes from @ThomasdenH's log file posted in  #1569

**Comment by LipuFei** [comment]
@whirm Ah, ok. This has been fixed, so I'm closing this one.

**Comment by whirm** [comment]
@LipuFei I'm confirming that this comes from an RC4 run. Let's leave it open until we are sure it's fixed.

**Comment by LipuFei** [comment]
Should be fixed by #1630

**Comment by whirm** [comment]
Not happening anymore on win 7 and win 10 with latest post-rc4 builds.
